ui/apps/dashboard/src/components/Layout/SideBar.tsx

```diff
@@ -1,26 +1,43 @@
 'use client';
 
-import { useState, type ReactNode } from 'react';
+import { useState } from 'react';
 
+import type { Environment } from '@/utils/environments';
+import { Help } from '../Navigation/Help';
+import { Integrations } from '../Navigation/Integrations';
 import Logo from '../Navigation/Logo';
+import Navigation from '../Navigation/Navigation';
+import { Profile } from '../Navigation/Profile';
 
 export default function SideBar({
   collapsed: serverCollapsed,
-  children,
+  activeEnv,
+  profile,
 }: {
   collapsed: boolean;
-  children: ReactNode;
+  activeEnv?: Environment;
+  profile: { orgName?: string; fullName: string };
 }) {
   const [collapsed, setCollapsed] = useState(serverCollapsed);
 
   return (
-    <div
-      className={`bg-canvasBase border-subtle top-0 flex h-screen flex-col justify-start ${
-        collapsed ? 'w-[64px]' : 'w-[224px]'
-      }  shrink-0 border-r transition-all delay-150 duration-300`}
+    <nav
+      className={`bg-canvasBase border-subtle
+         top-0 flex h-screen flex-col justify-start ${
+           collapsed ? 'w-[64px]' : 'w-[224px]'
+         }  sticky z-[500] shrink-0 overflow-visible border-r
+         `}
     >
       <Logo collapsed={collapsed} setCollapsed={setCollapsed} />
-      {children}
-    </div>
+      <div className="flex grow flex-col justify-between">
```

Comment:
All these items have been moved to "use client" so we can do active nav detection.